t/sanity.t

```diff
@@ -590,3 +590,59 @@ read: ["eof"]
 --- no_error_log
 [error]
 
+
```

Comment:
Style: we use 3 blank lines to separate test blocks (we usually use the [reindex](https://raw.githubusercontent.com/agentzh/old-openresty/master/bin/reindex) script to automatically format `.t` files for us).